runtime/interpreter/decode.go

```diff
@@ -1084,15 +1071,10 @@ func (d *DecoderV4) decodeCapability(v interface{}) (CapabilityValue, error) {
 
 	if field3 := encoded[encodedCapabilityValueBorrowTypeFieldKey]; field3 != nil {
 
-		decodedStaticType, err := d.decodeStaticType(field3)
+		borrowType, err = d.decodeStaticType(field3)
 		if err != nil {
 			return CapabilityValue{}, fmt.Errorf("invalid capability borrow type encoding: %w", err)
 		}
-
-		borrowType, ok = decodedStaticType.(StaticType)
```

Comment:
`Wondering why this was here, `decodeStaticType` already returns a `StaticType` even on `master`. I guess it used to return `interface{}`. Nice cleanup 👍